docs/releases/3.2.10.txt

```diff
@@ -10,4 +10,5 @@ Django 3.2.10 fixes a security issue with severity "low" and several bugs in
 Bugfixes
 ========
 
-* ...
+* Fixed a regression in Django 3.2 that caused a crash of ``setUpTestData()``
+  with ``BinaryField`` on PostgreSQL (:ticket:`33333`).
```

Comment:
Not knowing the details, the ticket specifically mentions `BinaryField`, but ... 
A bit of googling suggests that people have had issues with _pickling_ other fields which apparently contain memoryviews. I'd assume the fix handles that indiscriminately (if it's even an issue, given it's about using `deepcopy` rather than `pickle`), but I wonder if `BinaryField` needs calling out vs. built-in fields which are backed by a memoryview....

https://github.com/noripyt/django-cachalot/issues/115 (mentions `PolygonField`, `PointField`)
https://github.com/noripyt/django-cachalot/issues/125 (sort of suggests GIS in general, but I know nothing of GIS :))

Reply:
I considered handling only `memoryviews` for `BinaryFields` but I don't think it's worth the extra complexity. I also tried `PointField` after you comment and couldn't reproduce the same issue. IMO it's nice when release notes are concrete, `BinaryField` on PostgreSQL is the only confirmed case so it should be good to mention it explicitly.

I added a small note that `BinaryField` is memoryview-backed in this case. Do you think this could be helpful?

Reply:
Oh I definitely wasn't suggesting _only_ handling `memoryviews` for `BinaryField`. Indiscriminate is good in this case precisely because it should handle any other exotic fields which might involve a `memoryview`. Apologies if that was confusing.

My question was really around whether the other fields might have been quietly suffering from the same issue, and if so specifying them too (even though the _ticket_ was only about `BinaryField`). If the others don't exhibit the same problem, well that's just good news isn't it? :)

I appreciate your adding the note (as you say, it's nice when release notes are concrete), but I'm torn about it - `memoryview` feels almost like an implementation detail, but at the same time, it's the cause of the issue. I think it's _probably fine_ either way, your preference.